pkgs/development/compilers/llvm/16/clang/default.nix

```diff
@@ -0,0 +1,128 @@
+{ lib, stdenv, llvm_meta
+, monorepoSrc, runCommand
+, substituteAll, cmake, ninja, libxml2, libllvm, version, python3
+, buildLlvmTools
+, fixDarwinDylibNames
+, enableManpages ? false
```

Comment:
if it wasn't copy pasted, we should rename this to something like onlyManpages because usually this would enable the manpages in addition and not replacing the program

Reply:
If you want, you can apply this treewide for all relevant LLVM versions.